docs/user/servers/lib/LIB-MS.md

```diff
@@ -93,8 +93,125 @@ yarn start
 The lib microservice is now running and ready to serve files, functions, and models.
 
 You can access the server's endpoint by typing in the following URL: `http://localhost:<PORT>/lib`.
+
+````
+
+### Lib request and response
+
+Documentation of the lib query and responses.
+
 ```
 
+HTTP Request:
+
+.....
+
+send the request to: http://foo.com:<PORT>/lib
+
+POST /lib
+
+Host: foo.com:<PORT>
+
+Content-Type:application/json
+
+User-Agent:Mozilla
+
+Accept:_/_
+
+{
+
+"query":
+"query {
+  listDirectory {
+    project(fullPath: "/domain/") {
```

Comment:
The query given here is not reproducible. That's because the domain and parsed path are variables. Since this is an example, it's best to give concrete values that match the response given below.